lab-miketreat/lib/module.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const fs = require('fs');
+
+let addToBottomOfFile = function (filePath, text){
+  fs.readFile(`${__dirname}${filePath}`, function(err, data){
+    if(err) throw new Error('Error with fs.readFile in module.js line 6');
```

Comment:
Generally with async work, you'll want to pass errs from your helper functions to your callback.